### DISCUSSION TITLE
vue的仓库 没有dist文件夹

### DISCUSSION THREAD

**Discussion by serious-snow** [amll-dev/applemusic-like-lyrics#236] [General]
https://www.npmjs.com/package/@applemusic-like-lyrics/vue?activeTab=code
最新版本0.1.0

**Comment by Steve-xmh** [comment]
会尽快修正

**Comment by serious-snow** [comment]
看到发布了新版本，谢谢。
还有一个问题请教一下：
vue 和 react package.json 依赖的 @applemusic-like-lyrics/core 还是^0.0.15 
这个需要更新吗 ?

```  "dependencies": {
    "@applemusic-like-lyrics/core": "^0.0.15"
  },```

**Reply by serious-snow** [reply]
@Steve-xmh

**Comment by serious-snow** [comment]
@applemusic-like-lyrics/vue和react是不是用dev构建的哦，因为BackgroundRenderProps.albumImageUrl变成了album。
我升级后目前背景和歌词都无法工作。
版本如下：
```
    "@applemusic-like-lyrics/core": "^0.1.3",
    "@applemusic-like-lyrics/vue": "^0.1.5",
```
如果还原为0.0.15是正常的

**Reply by serious-snow** [reply]
@Steve-xmh 很抱歉一直@你。🙏

**Reply by Steve-xmh** [reply]
npm模块暂且先不要使用吧，我可能还需要一阵时间测试

**Reply by serious-snow** [reply]
好的，等你发布了新版我再试试

**Reply by Steve-xmh** [reply]
我重新看了一下目前发布的版本，我中途有给背景修改过接口，现在已经更名为 album 参数，然后允许直接传入图像/视频元素或者原先的链接字符串，如果需要传入视频还需要一同设置 albumIsVideo 属性。
检查一下你的依赖版本，react/vue绑定应该是 0.1.5，core应该是 0.1.3
